modules/tools/elasticsearch/main.tf

```diff
@@ -77,3 +77,42 @@ resource "helm_release" "elasticsearch_curator" {
     helm_release.elasticsearch
   ]
 }
+
+resource "kubernetes_manifest" "schedule_velero_elasticsearch_daily_backup" {
```

Comment:
Maybe we should make the schedule resource its own modules to set some defaults and reduce duplicate code?